services/profile/profileService.ts

```diff
@@ -0,0 +1,146 @@
+import axios, { AxiosInstance } from 'axios';
+import { Platform } from 'react-native';
+import { store } from '../../store/App.store';
+import { assert } from '../utils';
+import {
+  CreateProfileRequest,
+  CreateProfileResponse,
+  GetProfileRequest,
+  GetProfileResponse,
+  GetProfileFriendsRequest,
+  GetProfileFriendsResponse,
+  GetProfilePhotoRequest,
+  GetProfilePhotoResponse,
+  GetProfileChallengesResponse,
+  GetProfileFriendRequestsResponse,
+  SetProfilePhotoRequest,
+  SetProfilePhotoResponse,
+} from './types';
+
+declare global {
```

Comment:
I connected the frontend to the backend by matching how the Ross/Michael's tests handle the file input, which is via `req.files`. I had to finagle some things by making this module to get axios to accept our photo `FormData` object because I don't think TypeScript liked the typing for the default FormData types.